### ISSUE TITLE
Webpack & Vue.js

### ISSUE THREAD

**Issue by KlausSchaefersAtWork** [Azure-Samples/SpeechToText-WebSockets-Javascript#44]
We are trying to use the SDK with Webpack and ES6 and we get the same errors reported in issue (40 & 37). Although these issues were closed, the proposed solution do not work in in our setup (Vue.js with Webpack and ES6).

The main issue is that the SDK build is not following NPM (and commonJS) standards and the exports are not correctly exposed. Fixing these issues on customer level is the wrong solution.

You can find more information about the issue at:

https://github.com/Azure-Samples/SpeechToText-WebSockets-Javascript/issues/40
https://github.com/Azure-Samples/SpeechToText-WebSockets-Javascript/issues/37
 
Kind Regards,

Klaus

**Comment by raaaar** [comment]
What is the version of `microsoft-speech-browser-sdk` you have installed?

**Comment by KlausSchaefersAtWork** [comment]
The latest from npm

**Comment by raaaar** [comment]
The latest has a fix for #40. Haven't tried Webpack, but the sdk can be fairly trivially re-transpired for commonJS: 

```
diff --git a/gulpfile.js b/gulpfile.js
index 3cf1a78..4fb6384 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -27,8 +27,7 @@ gulp.task("build", function() {
             declaration: true,
             noImplicitAny: true,
             removeComments: true,
-            module: "AMD",
-            out: 'speech.browser.sdk.js'
+            module: "commonjs"
         }))
         .pipe(sourcemaps.write("."))
         .pipe(minify())
```

**Comment by salvoravida** [comment]
look @ https://github.com/Azure-Samples/SpeechToText-WebSockets-Javascript/pull/52

**Comment by raaaar** [comment]
Fixed in 606a619c54f1df81f0719f2fbde2ad674a0c8f33.
